src/test/kotlin/no/nav/helse/SøknadstypeTest.kt

```diff
@@ -37,6 +37,7 @@ internal class SøknadstypeTest {
             "ENABLE_OMSORGSPENGESKNAD_UTBETALING_ARBEIDSTAKER_ETTERSENDING" to "false",
             "ENABLE_OMSORGSPENGESKNAD_OVERFRING_AV_DAGER" to "false",
             "ENABLE_OMSORGSPENGEMELDING_DELING_AV_DAGER" to "false",
+            "ENABLE_OMSORGSPENGEMELDING_DELING_AV_DAGER_ETTERSENDING" to "false",
```

Comment:
Hadde vært nyttig å bruke unleash feature flag til dette f.eks.

Reply:
Ja det kunne vært et bruksområdet.

Reply:
Trenger ikke å ta det i denne PR-en, men noe å tenke på :)

Reply:
Hehe ja skjønte det :P Dette ble laget for å kunne skru av journalføring uten å stoppe prosesseringstjenestene? Bare blokkere endepunktet slik at det går i loop?

Reply:
Jepp.

Reply:
Burde det lages et "Issue" for det, slik at man ikke glemmer det helt?

Reply:
Kan være lurt 👍